TestCntlrApp/src/enbApp/nb_s1ap_hndlr.c

```diff
@@ -643,14 +643,8 @@ PRIVATE S16 nbBldS1SetupReq
       {
          tACItem = &ie->value.u.sztSuppTAs.member[taIdx];
          nbFillTknU8(&(tACItem->pres), PRSNT_NODEF);
-
```

Comment:
This change does not look compatible for testing the expected behavior of multi ENB test cases with different PLMN and TACs like "**s1aptests/test_multi_enb_multi_ue_diff_plmn.py**". It looks like it will always configure the same plmn and tac for all ENBs in multi ENB scenario. Can you please verify for  any of such test cases?

I am attaching here the successful pcap of above test case for comparison reference.
[multi_enb_diff_plmn_diff_tac.zip](https://github.com/magma/S1APTester/files/8703217/multi_enb_diff_plmn_diff_tac.zip)

Also if possible, please share the pcap for test case **s1aptests/test_multi_enb_multi_ue_diff_plmn.py** with this change

Reply:
The changes here only modify the logic when the `suppTA` parameter is passed from the test case. No existing tests use this parameter except for the new test that was added (`s1aptests/test_s1setup_success_secondary_plmn.py`)

This define never made sense as it was ignoring the PLMN that was specifically passed by the test case in `suppTA` if `MULTI_ENB_SUPPORT` was defined (which it is by default).

I am not able to provide a pcap at this time, but when these changes were originally made a month ago, I ran the entire suite of test cases to confirm that no tests were broken by this change.